### ISSUE TITLE
It would be very useful to clarify for each test c...

### ISSUE THREAD

**Issue by jensscheerlinck** [inspire-eu-validation/RETIRED-cross-cutting-issues#8]
_This issue has been extracted from the issue list on:https://ies-svn.jrc.ec.europa.eu/issues/2685_
# Comment

It would be very useful to clarify for each test case which messages should be reported in case of identified errors in the test object.
While the description of a test case should be independent from both the implementation and the values, the test case 'should be complete in the sense that it is sufficient to enable a test verdict to be assigned unambiguously to each potentially observable test outcome (i.e. sequence of test events)' (see ISO 19105). This level of completeness would imply that the potential verdicts / messages can be derived easily from the test case description. 
In some cases this is trivial, but in general there is a risk that the messages implemented in an ETS may not contain the information that the ATS authors expect.
# Proposed Change

For each test case, specify the messages (including the variable information derived from the test object) that should be reported.

**Comment by michellutz** [comment]
Agreed in principle. But we need to be aware that this could be considerable work.

One way to implement the proposed change would be to update the ATSs, when the error messages are implemented in the ETSs.

**Comment by cportele** [comment]
This approach sounds realistic to me.
